cedar-policy/src/api.rs

```diff
@@ -1338,7 +1340,7 @@ impl FromStr for Schema {
     /// found to not be a valid attribute name according to the Cedar
     /// grammar.
     fn from_str(schema_src: &str) -> Result<Self, Self::Err> {
-        Ok(Self(schema_src.parse()?))
+        Self::from_cedarschema_str(schema_src).map(|(schema, _)| schema)
     }
```

Comment:
This change makes sense to me, but I think the implementation for `SchemaFragment` still uses JSON. Make sure the two are consistent.

Reply:
Good catch.